tests/e2e/support/page/files/version.ts

```diff
@@ -0,0 +1,28 @@
+import { Actor } from '../../types'
+import { filesCta } from '../../cta'
+
+export class VersionPage {
+  private readonly actor: Actor
+  private readonly fileVersionSidebarXPath: string
+
+  constructor({ actor }: { actor: Actor }) {
+    this.actor = actor
+    this.fileVersionSidebarXPath = '//*[@id="oc-file-versions-sidebar"]/table/tbody/tr'
```

Comment:
and here we can do something like: `this.fileVersionSidebarXPath = page.locator('//*[@id="oc-file-versions-sidebar"]/table/tbody/tr')`